spec/integration/inventory_spec.rb

```diff
@@ -25,24 +25,45 @@ module Hawkular::Inventory::RSpec
     end
   end
 
-  describe 'Inventory', vcr: { decode_compressed_response: true } do
+  describe 'Inventory' do
     before(:all) do
       @creds = {
         username: 'jdoe',
         password: 'password'
       }
       @client = Hawkular::Inventory::InventoryClient.create(credentials: @creds)
-      @state = {
-        hostname: 'localhost.localdomain',
-        feed: nil
-      }
+      options = { decode_compressed_response: true }
+      options[:record] = :all if ENV['VCR_UPDATE'] == '1'
+      VCR.use_cassette('Inventory/Helpers/get_feeds', options) do
+        feeds = @client.list_feeds
+        @state = {
+          feed_uuid: feeds[0]
+        }
+      end
+    end
+
+    after(:all) do
+      require 'fileutils'
+      FileUtils.rm_rf "#{VCR.configuration.cassette_library_dir}/Inventory/tmp"
+    end
+
+    let(:cassette_name) do |e|
+      description = e.description
+      description
```

Comment:
What was the exact error/issue with the let(:cassette_name, &:description)` ? it might also be rspec version related.
Regardless, you don't *have* to follow all rubocop recommendations.